openpype/modules/deadline/plugins/publish/submit_maya_deadline.py

```diff
@@ -29,7 +29,8 @@
 
 import clique
 import requests
-
+from openpype.api import (
+    get_current_project_settings)
```

Comment:
Doesn't most other code in OP format it like this?
```suggestion
from openpype.api import (
    get_current_project_settings
)
```

Reply:
For some reason my linter panicked when I did that, fixing it.

Reply:
Done!